CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Backtrace Unity Release Notes
 
+## Version 3.2.5
+- Added `BacktraceClient` Initialization method that allows developer to intialize Backtrace integration without adding game object to game scene.
+- Fixed invalid `meta` file for iOS integration for Unity 2019.2.13f1.
+- Backtrace client allows to configure report sampling. By default report sampling is equal to 0.01 - which means 1% of all unhandled exceptions will be skipped. 
```

Comment:
The expectation here is that 1% of unhandled exceptions will be reports. Worth also clarifying that it is random sampling.

```
- Enables a new random sampling mechanism for unhandled exceptions.
```

Reply:
Clarify this information in changelog and readme. Thanks